config/windows/build.xml

```diff
@@ -90,7 +90,7 @@
 		<mkdir dir="${bin.native}/build"/>
 
 		<!-- CORE -->
-		<build name="lwjgl" dest="${bin.native}/core">
+		<build name="lwjgl" dest="${bin.native}/core" flags="/wd4710">
```

Comment:
This disables all warnings and treat-warnings-as-errors for the entire LWJGL core. It's better to isolate this in `MemoryAccess.c`, using a pragma like:

```c
#ifdef LWJGL_WINDOWS
	__pragma(warning(disable : 4710))
#endif
```